<launch>
    <param
        name="DirectionButtons/serial/device"
        type="string"
        value="/dev/input/js0"
     />

    <param
        name="DirectionButtons/direction/forward"
        type="string"
        value="255,127,2,1"
     />
    <param
        name="DirectionButtons/direction/forwardRelease"
        type="string"
        value="0,0,2,1"
     />
    <param
        name="DirectionButtons/direction/backward"
        type="string"
        value="1,128,2,1"
     />
    <param
        name="DirectionButtons/direction/backwardRelease"
        type="string"
        value="0,0,2,1"
     />
    <param
        name="DirectionButtons/direction/left"
        type="string"
        value="255,127,2,0"
     />
    <param
        name="DirectionButtons/direction/leftRelease"
        type="string"
        value="0,0,2,0"
     />
    <param
        name="DirectionButtons/direction/right"
        type="string"
        value="1,128,2,0"
     />
    <param
        name="DirectionButtons/direction/rightRelease"
        type="string"
        value="0,0,2,0"
     />

    <param
        name="DirectionButtons/buttons/start"
        type="string"
        value="1,0,1,3"
     />
    <param
        name="DirectionButtons/buttons/startRelease"
        type="string"
        value="0,0,1,3"
     />
    <param
        name="DirectionButtons/buttons/top"
        type="string"
        value="1,0,1,1"
     />
    <param
        name="DirectionButtons/buttons/topRelease"
        type="string"
        value="0,0,1,1"
     />
    <param
        name="DirectionButtons/buttons/middle"
        type="string"
        value="1,0,1,2"
     />
    <param
        name="DirectionButtons/buttons/middleRelease"
        type="string"
        value="0,0,1,2"
     />
    <param
        name="DirectionButtons/buttons/bottom"
        type="string"
        value="1,0,1,0"
     />
    <param
        name="DirectionButtons/buttons/bottomRelease"
        type="string"
        value="0,0,1,0"
     />

    <node
        pkg="DirectionButtons"
        name="DirectionButtons"
        type="DirectionButtons.py"
        args=""
        output="log"
        respawn="false"
     />
</launch>
